test/broken-links.md

```diff
@@ -2,7 +2,9 @@
 title: Broken Link Test
 subtitle: Version X
 author:
+
```

Comment:
We should not list frontmatter as Markdown but as YAML.

Reply:
Unnecessary YAML frontmatter changes have been removed.